cmd/run.go

```diff
@@ -146,6 +147,8 @@ func start(cliCtx *cli.Context) error {
 		EventID:    event.EventID_NodeComponentStarted,
 	}
 
+	var poolInstance *pool.Pool
+
```

Comment:
This change would force all the components running at the same binary to concur the db access. Multiple components will run different go routines but will consume a single pool Instance. 

it's ok to initialize multiple pool instances for different components and the time it takes do instantiate shouldn't be a problem, I don't see a reason for this optimization and I'm afraid of the side effects it can cause when talking about concurrent and parallel executions to the pool.

What is the rational behind this change? Have you faced an issue during the development?

Reply:
I just thought it was better to just have one pool per binary. I can undo this is you don't feel confortable, or we are not sure if it may cause problems.

Reply:
Single pool, maybe better performance, maybe side cases.
Multiple pools, work as always, and already tests multiple times.

It's up to you, my suggestion is that we shouldn't touch on a team that's winning unless you want more emotion in your life 🤣